packages/client/components/ReflectionGroup/RemoteReflection.tsx

```diff
@@ -103,16 +106,41 @@ const getHeaderTransform = (ref: RefObject<HTMLDivElement>, topPadding = 18) =>
 const getInlineStyle = (
   remoteDrag: RemoteReflection_reflection['remoteDrag'],
   isDropping: boolean | null,
-  style: React.CSSProperties
+  style: React.CSSProperties,
+  showAboveSpotlight: boolean
 ) => {
   if (isDropping || !remoteDrag || !remoteDrag.clientX) return {nextStyle: style}
-  const {left, top, minTop} = getCoords(remoteDrag as any)
-  return {nextStyle: {transform: `translate(${left}px,${top}px)`}, minTop}
+  const {left, top, minTop} = getCoords(remoteDrag as any, showAboveSpotlight)
+  const zIndex = showAboveSpotlight ? ZIndex.REFLECTION_IN_FLIGHT_SPOTLIGHT : style.zIndex
+  return {nextStyle: {transform: `translate(${left}px,${top}px)`, zIndex}, minTop}
+}
+
+const getTarget = (meetingId: string, targetId: string, atmosphere: Atmosphere) => {
+  let isTargetInSpotlight = false
+  commitLocalUpdate(atmosphere, (store) => {
```

Comment:
We're not updating the store here - just grabbing a value. I prefer using the Relay store instead of adding another Root query as it saves a trip to the server, and getting the required `reflectionId` and `searchQuery` variables for a Root query can be ugly. 

In [ErrorBoundary](https://github.com/ParabolInc/parabol/blob/89f7d0cceae2c30d7e8757767418ddade4ae98c1/packages/client/components/ErrorBoundary.tsx), I see we use `atmosphere.getStore()`, cast the `store` to `any`, and can then access the records, but I think this is slightly easier to read

Reply:
Try useQueryLoader: https://relay.dev/docs/guided-tour/reusing-cached-data/fetch-policies/